src/outputs/export_files.py

```diff
@@ -0,0 +1,177 @@
+"""This is a stand alone pipeline to selectively transfer output files from
+the output folder to the outgoing folder, along with their manifest file."""
+
+
+import os
+import logging
+from datetime import datetime
+import toml
+
+from src.utils.helpers import Config_settings
+from src.outputs.manifest_output import Manifest
+
+
+# Set up logging
+OutgoingLogger = logging.getLogger(__name__)
+
+
+config_path = os.path.join("src", "developer_config.yaml")
+
+# load config
+conf_obj = Config_settings(config_path)
+config = conf_obj.config_dict
+
+# Check the environment switch
+network_or_hdfs = config["global"]["network_or_hdfs"]
+
+if network_or_hdfs == "network":
+
+    from src.utils.local_file_mods import local_list_files as list_files
+    from src.utils.local_file_mods import local_copy_file as copy_files
+    from src.utils.local_file_mods import local_move_file as move_files
+    from src.utils.local_file_mods import local_search_file as search_files
+    from src.utils.local_file_mods import local_isfile as isfile
+    from src.utils.local_file_mods import local_delete_file as delete_file
+    from src.utils.local_file_mods import local_md5sum as hdfs_md5sum
+    from src.utils.local_file_mods import local_stat_size as hdfs_stat_size
+    from src.utils.local_file_mods import local_isdir as isdir
+    from src.utils.local_file_mods import local_read_header as read_header
+    from src.utils.local_file_mods import (
+        local_write_string_to_file as write_string_to_file,
+    )
+
+elif network_or_hdfs == "hdfs":
+
+    from src.utils.local_file_mods import hdfs_list_files as list_files
+    from src.utils.hdfs_mods import hdfs_copy_file as copy_files
+    from src.utils.hdfs_mods import hdfs_move_file as move_files
+    from src.utils.hdfs_mods import hdfs_search_file as search_files
+    from src.utils.hdfs_mods import hdfs_isfile as isfile
+    from src.utils.hdfs_mods import hdfs_delete_file as delete_file
+    from src.utils.hdfs_mods import hdfs_md5sum as hdfs_md5sum
+    from src.utils.hdfs_mods import hdfs_stat_size as hdfs_stat_size
+    from src.utils.hdfs_mods import hdfs_isdir as isdir
+    from src.utils.hdfs_mods import hdfs_read_header as read_header
+    from src.utils.hdfs_mods import hdfs_write_string_to_file as write_string_to_file
+
+
+else:
+    OutgoingLogger.error("The network_or_hdfs configuration is wrong")
+    raise ImportError
+
+OutgoingLogger.info(f"Using the {network_or_hdfs} file system as data source.")
+
+
+# Define paths
+NETWORK_OR_HDFS = config["global"]["network_or_hdfs"]
+paths = config[f"{NETWORK_OR_HDFS}_paths"]
```

Comment:
You've converted this to lower case in other places. Maybe the same applies here

Reply:
Fixed